base/applications/network/dwnl/dwnl.c

```diff
@@ -51,17 +54,19 @@ write_status(LPCTSTR lpFmt, ...)
 
     if (GetConsoleScreenBufferInfo(GetStdHandle(STD_OUTPUT_HANDLE), &csbi))
```

Comment:
Note to /self: similar stuff is already dealt by conutils.

Reply:
Is this something that I'm supposed to do anything with?

Reply:
Don't worry, keep the code as it is now, I'll adapt it later if really needed.